css/about.css

```diff
@@ -0,0 +1,109 @@
+html, body {
+    
+     height: 100%;
+    width:800px; */
+    font-family: 'Roboto', sans-serif;
+    margin: 0 0 0 60px;
+}
+
+container {
+	margin: 0 auto;
+	/* width: 800px; */
+}
+
+* {
+    box-sizing: border-box;
+}
+a{
+    text-decoration: none;   
+}
+
+h1, h2, h3, h4, h5 {
+    font-size: 18px;
+    margin-bottom: 15px;
+    font-family: 'Rubik', sans-serif;
+}
+
+.navBar{
+    height:80px;
+    display:flex;
+    flex-direction:row-reverse;
+    justify-content:space-between;
+    align-items: center;
+    margin: 0 30px 0 0;
+    
+}
+.navBar{
+  
+    height:80px;
+    display:flex;
+    flex-direction:row-reverse;
+    justify-content:space-between;
+    align-items: center;
+    margin: 0 30px 0 0;
+    
+}
+.navBar nav{
+    Width: 75%;
+    display:flex;
+    justify-content:space-between;
+    align-items: baseline;
+    padding: 20px 0 0 0;
+    color: black;
+}
+.navBar img {
+    display:flex;
+    justify-content:space-between;
+    align-items: center;
+}
```

Comment:
Although this may appear insignificant, it's important to be consistent when spacing, such as whether to have a space after the `:` or not.